manticore/manticore.py

```diff
@@ -168,7 +168,7 @@ def __init__(self, path_or_state, argv=None, workspace_url=None, policy='random'
         self.forward_events_from(self._executor)
 
         if isinstance(path_or_state, str):
-            assert os.path.isfile(path_or_state)
+            assert os.path.isfile(path_or_state), "The file or state %s doesn't exist" % path_or_state
```

Comment:
could we raise an `Exception` instead for this, with that error message? I think that would be a little more appropriate than an assertion failure, and a little more pythonic. ideally we wouldn't raise a plain `Exception`, but we haven't gotten around to creating a Manticore exception hierarchy, so for the time being, I think `Exception` will be fine?

Reply:
@mossberg Yeah, `Exception`s are much better. We can use `Exception` for now. Fixed.